### PR TITLE
feat: use real data for the base line plot

### PR DESCRIPTION
**Pull request by guadiromero**
Changes introduced in this PR:
- Use real data instead of dummy data for the base line plot
- Use a different y range for each unit of measure, so the lines distribute better along the graph
- Start the x range at 0
- Do not use scientific notation for the y axis labels
- Show all the x axis labels

Note: There is still some refactoring and minor style changes to come in future PRs.

### CODE REVIEW

**Review by herrherrmann — APPROVED** [review]
I guess for the line charts we eventually want to have more "human-friendly" text for the labels (translatable Title Case instead of snake_case). But that’s probably a change for the future?

Bildschirmfoto 2023-11-13 um 15 58 45

Otherwise, this works and looks good to me!

**Comment by guadiromero** [comment]
> I guess for the line charts we eventually want to have more "human-friendly" text for the labels (translatable Title Case instead of snake_case). But that’s probably a change for the future?
> 
> Bildschirmfoto 2023-11-13 um 15 58 45
> 
> Otherwise, this works and looks good to me!

Yes, this will come later (details about titles, labels, colors) :) Shouldn't take much effort.

File: app/sliders/pn_app.py [hunk 1]
```diff
@@ -78,7 +78,9 @@
 
 def get_base_chart():
     # TODO: Import real data and adjust plotter accordingly
```
[the remark this answers is not in the document]
Comment (guadiromero) [reply app/sliders/pn_app.py:80]: Genau, will take care of it

File: app/sliders/config_custom.yaml [hunk 2]
```diff
@@ -31,13 +30,31 @@ base_line_interactive:
     - de
   filters:
     single_choice:
-      - anzahl
-      - umsatz
-    single_choice_default: anzahl
+      - anzahl_unternehmen_insgesamt
+      - anzahl_innovative_unternehmen
+      - anzahl_innovatoren
+      #- umsatz_mio
```
[the remark this answers is not in the document]
Comment (guadiromero) [reply app/sliders/config_custom.yaml:36]: I commented it on purpose. The values are so much higher with this unit of value that the plot ends up looking not very good. We have to decide how to display it (or if we want to leave it out) with the rest of the team. In general, the config file is the one file that is supposed to be tweaked quite freely depending on what we want to plot.

**Pull request merged**